portfolio/src/main/webapp/index.html

```diff
@@ -3,10 +3,18 @@
   <head>
     <meta charset="UTF-8">
     <title>Aileen's Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="style.css" />
     <script src="script.js"></script>
+
+    <script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
+    <script
+      src="https://maps.googleapis.com/maps/api/js?key=AIzaSyD6EPpSPCTfaYS8Qh9HXFjNc14GDF7DYR4&callback=initMap&libraries=&v=weekly"
+      defer
+    ></script>
```

Comment:
[nit] I would move the closing ">" to line 12, and have line 13 be just </script>